service/src/main/resources/db/changesets/20220923_initial.yaml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: initial
+      author: ishaghag
+      changes:
+        - createTable:
+            tableName: user_profile
+            columns:
+              - column:
+                  name: user_id
+                  type: text
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+                  remarks: A persistent user ID retrieved from SAM
+              - column:
+                  name: profile_obj
+                  type: jsonb
+                  constraints:
+                    nullable: false
+                  remarks: Object storing the profile data
```

Comment:
Could we also consider adding a 'metadata' object to all tables going forward? 
This would be an internal field that can be used for logs / debug

some of the fields could be - 

- version text // object version
- created_by, updated_by text // which client / app / job led to this record
- created_at, updated_at timestamp // may not be needed, if this can be fetched from DB logs
- is_deleted boolean // potential future use if we want to support soft delete

Reply:
Agree, Since this is a new concept that is being proposed for Terra service DBs, we can take it outside the current PR (not a blocker).
I will take this proposal offline